app/services/cache/ExportsCacheModelRepository.scala

```diff
@@ -69,6 +70,8 @@ class ExportsCacheModelRepository @Inject()(mc: ReactiveMongoComponent, appConfi
         updateResult.result[ExportsCacheModel]
       }
 
+  def remove(sessionId: String): Unit = collection.findAndRemove(selector = bySessionId(sessionId)).map(_ => ())
```

Comment:
Why we ignoring result future with this - even logging of errors ?

Reply:
Have changed how we do this so all code in controller is handled inside the future.